app/code/Magento/Checkout/view/frontend/web/js/model/error-processor.js

```diff
@@ -29,6 +29,7 @@ define([
                 try {
                     error = JSON.parse(response.responseText);
                 } catch (exception) {
+                    console.error(exception);
                     error = {
```

Comment:
the question here we actual want to expose errors to console may cause vulnerable web application ?